### ISSUE TITLE
ValueError: cannot reshape array of size 283154 into shape (256,128,3,3) on weights with one class

### ISSUE THREAD

**Issue by KacperPaszkowski** [zzh8829/yolov3-tf2#324]
I'm trying to convert weights trained on one class and I get error:

> ValueError: cannot reshape array of size 283154 into shape (256,128,3,3)
on layer 3296 utils.py:45] yolo_output_2/conv2d_73 bn while using command:

> python convert.py --weights ./data/yolov3.weights --num_classes 1 --output ./checkpoints/yolov3.tf

**Comment by ofeksadlo** [comment]
Make sure the number of classes is right I had the same issue
and that was the problem.

**Comment by KacperPaszkowski** [comment]
Yep. I have only 1 class. I tried with diffrent nubers of classes and result is always the same.

**Comment by rubetrio** [comment]
I'm hitting the same error and I did pass in the correct number of classes thru --num_classes

"ValueError: cannot reshape array of size 238038 into shape (256,128,3,3)"

**Comment by emichester** [comment]
Modify the line 11 on [convert.py](https://github.com/zzh8829/yolov3-tf2/blob/master/convert.py) to:

```python
flags.DEFINE_integer('num_classes', 1, 'number of classes in the model')
```
